playbooks/roles/insights_tracking_logs_sync/files/download_tracking_logs.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python
+
+import argparse
+import datetime
+import logging
+import math
+import os
+import shutil
+import socket
+import subprocess
+import sys
+
+import boto
+import raven
+
+
+def download_from_gcloud_storage(tracking_logs_dir, bucket):
+    """
+    Download a file from the specified Google Cloud Storage bucket.
+
+    Note that the host machine must be properly configured to use boto with a
+    Google Cloud Platform service account. See
+    https://cloud.google.com/storage/docs/xml-api/gspythonlibrary.
+
+    tracking_logs_dir: An absolute path to the dir where the files
+                       should stored.
+    bucket: The name of a Google Cloud Storage bucket.
+    """
+
+    import gcs_oauth2_boto_plugin
+
+    gcloud_uri = boto.storage_uri(bucket, 'gs')
+    for key in gcloud_uri.get_bucket():
+    	if not os.path.exists(tracking_logs_dir + key.name):
+    	    logging.info('Downloading tracking log "{}" from Google Cloud '
+                         'Storage bucket "{}"'.format(key.name, bucket))
+    	    tl_file = open(tracking_logs_dir + key.name, "w")
+            key.get_contents_to_file(tl_file)
```

Comment:
Goof find @mrgnr! Has this been tested? Also, how many items can  we expect to be in the bucket?